app/javascript/components/Results/BestBets/GraphicsBestBet.tsx

```diff
@@ -42,20 +42,20 @@ export const GraphicsBestBet = ({ title, titleUrl, imageUrl, imageAltText, links
     <GridContainer className='result search-result-item graphics-best-bets featured-collection'>
       <Grid row gap="md">
         {imageUrl && (
-          <Grid mobileLg={{ col: 4 }} className='result-thumbnail'>
+          <Grid mobileLg={{ col: 2 }} className='result-thumbnail'>
             <img src={imageUrl} alt={imageAltText && imageAltText} className="result-image"/>
           </Grid>
         )}
         <Grid col={true} className='result-meta-data'>
           <div className='graphics-best-bets-title result-title'>
             {titleUrl ? (
-              <a href={titleUrl} className='result-title-link'><h2 className='result-title-label'>{parse(title)}</h2></a>) : <h2 className='result-title-label'>{parse(title)}</h2>}
+              <h2 className='result-title-label'><a href={titleUrl} className='result-title-link'>{parse(title)}</a></h2>) : <h2 className='result-title-label'>{parse(title)}</h2>}
           </div>
           {links && links.length > 0 && (
             <Grid row gap="md">
               {sortedLinks(links).map((link, index) => {
                 return (
-                  <Grid key={index} mobileLg={{ col: (index as number) % 2 === 0 ? 7 : 5 }} className='graphics-best-bets-link-wrapper'>
```

Comment:
SRCH-4851: Fix for Best bets graphics title and link should be closer to the image - giving equal width to links